benchmark/fluid/fluid_benchmark.py

```diff
@@ -131,6 +131,10 @@ def train(avg_loss, infer_prog, optimizer, train_reader, test_reader, batch_acc,
     exe = fluid.Executor(place)
     exe.run(startup_prog)
 
+    # Use inference_transpiler to speedup
+    t = fluid.InferenceTranspiler()
+    t.transpile(infer_prog, place)
```

Comment:
It's not suitable to add transpiler function here, since its a `train` benchmark. @typhoonzero Do you have some better suggestion?

Reply:
I could move it higher to `main`. However, `InferenceTranspiler` requires place as a parameter and extracting it happens in `train`. So the drawback would be that I would have to extract place again. Or I could add 'place' parameter to `train` and extract it in `main.

Reply:
The benchmark of inference is on the original model now. Thus, how about remove line 134-136, or add an option to control it?